main.tf

```diff
@@ -82,8 +83,9 @@ resource "google_service_account" "org_terraform" {
 resource "google_storage_bucket" "org_terraform_state" {
   project                     = module.seed_project.project_id
   name                        = local.state_bucket_name
-  location                    = var.default_region
+  location                    = local.storage_bucket_location
   labels                      = var.storage_bucket_labels
+  storage_class               = var.storage_bucket_storage_class
```

Comment:
For storing the TF state I would think that most users would use `standard`. While we can accept this as an input with default, is there a strong reason for other classes?

Reply:
Most midsize to larger business customers prefer or expect the extra assurance of multi-regional.  Given the size of the data and associated costs, I think even the smallest companies should simply just expect it to be that way.  "earthquake proof" or whatever?  Kind of silly in some ways since typically we never need to run this module again other than the second time to move the actual bootstrap state up to the bucket from local.  Meanwhile - customer perceptions matter.

At a practical level (at least a pattern I frequently use) - many of the outputs from here are very useful in subsequent/subordinate terraform modules. Instead of having to define/specify a simple variable like 'org_id' repeatedly each and every time - just grab it from 'bootstrap' (w/read-only access).  Which of course creates a dependency on the availability of this bucket elsewhere.

Reply:
@michaeldeman totally, I think it makes sense to be multi-regional. I initially got confused btw `STANDARD ` and `MULTI_REGIONAL` exposed via providers.